pyarlo/base_station.py

```diff
@@ -113,7 +301,11 @@ def mode(self, mode):
         """
         if mode not in ACTION_MODES.keys():
             return "Invalid mode"
-        return self.__run_action(mode)
+        return self.__run_action(
+            method='SET',
+            resource='modes',
+            mode=mode,
+            publish_response=True)
```

Comment:
I have anyway called the update() function all doing the run_action() just to be sure.  I just wanted to avoid returning the wrong status !